Sources/CocoaLumberjack/DDOSLogger.m

```diff
@@ -13,105 +13,144 @@
 //   to endorse or promote products derived from this software without specific
 //   prior written permission of Deusty, LLC.
 
+#import <TargetConditionals.h>
 #import <os/log.h>
 
 #import <CocoaLumberjack/DDOSLogger.h>
 
-@interface DDOSLogger () {
-    NSString *_subsystem;
-    NSString *_category;
+@implementation DDOSLogLevelMapperDefault
+
+- (instancetype)init {
+    self = [super init];
+    return self;
+}
+
+- (os_log_type_t)osLogTypeForLogFlag:(DDLogFlag)logFlag {
+    switch (logFlag) {
+        case DDLogFlagError:
```

Comment:
How about mapping `DDLogFlagError` to [OS_LOG_TYPE_FAULT](https://developer.apple.com/documentation/kernel/os_log_type_t/os_log_type_fault) ?

Reply:
These kind of changes is exactly what the new level mapping facility can be used for.
I'd like to stick as close to the defaults here as possible, and since [`Logger.error` logs with the `error` level](https://developer.apple.com/documentation/os/logger/3551616-error), I'd do the same for our error log method.
As you correctly noted, to add `fault`, we'd need a level above `error`, which we currently don't have.